### DISCUSSION TITLE
Understanding how often findings are sent to security hub?

### DISCUSSION THREAD

**Discussion by qadri99-max** [prowler-cloud/prowler#5313] [Q&A]
1 - I want to understand a bit more how findings are sent to the Security Hub. For my scans I can see a log entry that says: '0 findings sent to AWS Security Hub'. 

Does that mean that if the same findings are found, Prowler does not send the same finding again to Security Hub? 

Under Security Hub, I can see findings that says Updated at: 7 days ago

2 - Can you please explain what does this mean:
"31m30.91% (484) Failed [0m]....."

thanks
![security-hub-02](https://github.com/user-attachments/assets/7ec0660b-7001-44f9-ac7f-1179a9ce4dee)

**Comment by jfagoagas** [comment]
Hello @qadri99-max again, as with the other discussion, I apologise for the late notice.

Regarding your questions:
1. By default all findings are sent to AWS Security Hub if you input the flag to enable it. Are you doing it? Also by default we archive the previous findings if they are not present in the new Scan to reduce the results since if they are not present means that those findings are no longer an issue.

What version of Prowler are you using?

3. These characters are the colors used in the CLI which are not rendered by AWS CloudWatch in their logs. There is a workaround [here](https://github.com/prowler-cloud/prowler/issues/3340#issuecomment-2141854585) to remove those values `prowler <...> | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2};?)?)?[mGK]//g".`. By the way we have a feature request to remove the color output from the CLI https://github.com/prowler-cloud/prowler/issues/5294 so you can keep track of it there.


Thanks for using Prowler 🚀
